CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ Release channels have their own copy of this changelog:
 #### Changes
 * `withdraw-stake` now accepts the `AVAILABLE` keyword for the amount, allowing withdrawal of unstaked lamports (#4483)
 * `solana-test-validator` will now bind to localhost (127.0.0.1) by default rather than all interfaces to improve security. Provide `--bind-address 0.0.0.0` to bind to all interfaces to restore the previous default behavior.
+* `simulateTransaction` now includes `loadedAccountsDataSize` in its result that is total bytes for loading all accounts in this transaction.
```

Comment:
This line is under the `CLI` section; I think `Validator` section would be more appropriate or even adding an `RPC` category

Reply:
Sorry, one more nit; I think this should be two sentences. Current one is a bit of a run-on IMO; maybe something like
> `simulateTransaction` now includes `loadedAccountsDataSize` in its result. `loadedAccountsDataSize` is the total number of bytes loaded for all accounts in the simulated transaction.